.github/workflows/lint-with-vale.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Linting with Vale on pull request
+on: [pull_request]
+
+jobs:
+  vale:
+    name: Linting with Vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@reviewdog
+        with:
+          filter_mode: diff_context
+          vale_flags: "--no-exit --minAlertLevel=error --glob=!*/doc-*/common/*"
+          # github-pr-annotations doesn't require elevated token privileges
+          reporter: github-pr-annotations
+          fail_on_error: true
```

Comment:
It has annotations, but the check doesn't appear to fail. That's confusing to me.